rqd/Dockerfile

```diff
@@ -52,6 +52,10 @@ RUN versioned_name="rqd-$(cat ./VERSION)-all" \
   && tar -cvzf $versioned_name.tar.gz $versioned_name/* \
   && ln -s $versioned_name rqd
 
+RUN mkdir -p /etc/opencue
+RUN echo "[Override]" >> /etc/opencue/rqd.conf
```

Comment:
nit: We should probably use `>` instead of `>>` here, so the file is overwritten if it already exists.

Not sure what happens if we end up with multiple `[Override]` sections in the config file -- probably it will break the config parser.

Reply:
> We should probably use > instead of >> here, so the file is overwritten if it already exists.

Noted. Addressed in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1311/commits/1e3b05b39fb35e9ddb3b71a75c441f0713271d62

> Not sure what happens if we end up with multiple [Override] sections in the config file -- probably it will break the config parser.

This happened to me when implementing the ENV PATH parameter in the RQD Blender dockerfile. Yes, the parser breaks, and none of the configs get applied.